blue-green-spring/files/imagestreams/imagestreams.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
There is an assumption that this ImageStream already exists in the cluster. Otherwise running these examples would require cluster-admin privileges